testing/reconciler.go

```diff
@@ -82,6 +82,8 @@ type ReconcilerTestCase struct {
 	ExpectedResult controllerruntime.Result
 	// Verify provides the reconciliation Result and error for custom assertions
 	Verify VerifyFunc
+	// VerifyCtx provides the context for custom assertions
+	VerifyCtx VerifyCtxFunc
```

Comment:
what's the use case for this on ReconcilerTestCase?

Reply:
Right! I forgot that `ReconcilerTestCase` doesn't expose stash-related assertions.